src/stylesheets/core/mixins/_icon.scss

```diff
@@ -154,9 +154,9 @@
 @mixin place-icon(
   $icon-object,
   $direction,
-  $margin,
-  $vertical-align,
-  $contrast-bg
+  $margin: "auto",
+  $vertical-align: middle,
+  $contrast-bg: "default"
```

Comment:
# `place-icon()` and `add-color-icon()` Defaults

Setting default values for these functions so users can simply set the icon object and direction while leaving the option to change the rest for optional specificity.